### PR TITLE
AQCU-1619: update pom to allow artifacts to upload to artifactory

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,19 +16,32 @@
...
+    </distributionManagement>
+    
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
         <version>1.5.10.RELEASE</version>
     </parent>
 
-    <repositories>
+    <!--repositories>
```
Comment (mhines-usgs) [inline pom.xml]: @zmoore-usgs do i have to comment each line like your PR or should this work?

File: pom.xml [hunk 2]
```diff
@@ -172,7 +186,7 @@
 
     <scm>
         <connection>scm:git:https://github.com/USGS-CIDA/java-to-r.git</connection>
-        <developerConnection>scm:git:https://github.com/USGS-CIDA/java-to-r.git</developerConnection>
+        <developerConnection>scm:git:https://${env.GITHUB_USER}:${env.GITHUB_PASS}@github.com/USGS-CIDA/aqcu-framework.git</developerConnection>
```
Comment (zmoore-usgs) [inline pom.xml]: This should change back to `java-to-r.git`

File: pom.xml (hunk 1 above)
Comment (zmoore-usgs) [reply pom.xml]: I think that should work!

Comment (mhines-usgs) [reply pom.xml]: cool!

File: pom.xml (hunk 2 above)
Comment (mhines-usgs) [reply pom.xml]: Woops, yep

**Pull request merged**